beacon-chain/p2p/utils.go

```diff
@@ -173,3 +179,26 @@ func verifyConnectivity(addr string, port uint, protocol string) {
 		}
 	}
 }
+
+func ConvertPeerIDToNodeID(pid peer.ID) (enode.ID, error) {
```

Comment:
Should we add a godoc comment?

Reply:
Sure, done in 85264bb21141ab238497fb23e985631d0f602799.